SpatialGDK/Documentation/content/technical-overview/gdk-concepts.md

```diff
@@ -14,32 +14,32 @@ Key concepts in Unreal Engine map to concepts in the GDK, as shown in the table
 | RPC | SpatialOS component’s command or event | A type of data stored in a SpatialOS component. Note that a SpatialOS component’s command is **not** the same as an Unreal command. |
 | Owning connection | EntityACL component | |
 | Conditional property replication | Dynamic `component_delivery` filter | |
-| Server | Server-worker instance | You can have multiple server-worker instances running the cloud element of your game. |
+| Server | Server-worker instance | You can have one or more server-worker instances running the cloud element of your game. |
 
 You can find out more about [entities]({{urlRoot}}/content/glossary#entity), SpatialOS [components]({{urlRoot}}/content/glossary#spatialos-component) and their properties, commands, and events, as well as [server-workers]({{urlRoot}}/content/glossary#worker), in the [glossary]({{urlRoot}}/content/glossary).
 
 ## GDK for Unreal concepts
 We’ve introduced some new concepts to facilitate the fact that SpatialOS enables you to spread computation between multiple servers - known as “server-worker instances” in SpatialOS.
 
 ### Offloading
-Offloading is the new architecture that the SpatialOS GDK for Unreal provides to allocate the authority of specific Actor groups from the main Unreal server worker instance to a different worker instance. By using offloading, you can save the resources of the main Unreal server-worker instance when you want to build richer game features.
+Offloading is one of the multiserver options for working with SpatialOS. It involves allocating the authority over specific groups of Actors to a server-worker instance other than your main Unreal server-worker instance. 
```

Comment:
I think this doesn't really explain what offloading is.
Have made a suggestion elsewhere on how to explain what it is.